static/docs/commands-reference/get.md

```diff
@@ -18,19 +18,26 @@ positional arguments:
 
 ## Description
 
-DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another DVC repository into the current
-working directory, regardless of whether it's a DVC project. The `dvc get`
-command downloads such a <abbr>data artifact</abbr>.
-
-The `url` argument specifies the address of the Git repository containing the
-external <abbr>DVC project</abbr> (both HTTP and SSH protocols supported, e.g.
-`[user@]server:project.git`). `path` is used to specify the path of the data to
-be downloaded within the repo.
+Provides an easy way to reuse datasets, intermediate results, ML models, or
+other files and directories (any <abbr>data artifact</abbr>) tracked in another
+DVC repository, by downloading them into the current working directory.
 
 Note that this command doesn't require an existing DVC project to run in. It's a
 single-purpose command that can be used out of the box after installing DVC.
 
+The `url` argument specifies the address of the Git repository containing the
+external <abbr>project</abbr>. Both HTTP and SSH protocols are supported for
+online repositories (e.g. `[user@]server:project.git`). `url` can also be a
+local file system path to an "offline" repository.
+
+The `path` argument of this command is used to specify the location of the data
+to be downloaded within the source project. It should point to a data file or
+directory tracked by that project – specified in one of the
```

Comment:
> Still a bit too complicated and hard to read tbh... not sure we need stuff about visibility...

@shcheklein This I had previously revised myself but just did some more improvements in 39a19f9.

> On the other hand, no matter how well explained, the commands get and import are intrinsically a bit confusing...

At least the arguments are somewhat hard to explain for sure, even if they do make sense but as mentioned before there should probably be a command to list data artifacts in an external project. I don't remember if there's an issue or discussion about it open still...

Reply:
> May it's better to show a simple code snippet...
> ```
> $ dvc get example-get-started model.pkl
> ```

I agree we shouldn't shy away from short command examples in descriptions but that is already the exact first example so the user only has to scroll a little bit to find it. I would use a different command but actually I don't think this command helps explain the fact that `path` is not actually a file in the Git repo, but a record in one of its DVC-files.

Reply:
> there should probably be a command to list data artifacts in an external project

Opened feature request in https://github.com/iterative/dvc/issues/2509.